README.md

```diff
@@ -247,3 +233,45 @@ Here are some additional reporters that are available from the hapijs community:
 - [good-loggly](https://github.com/fhemberger/good-loggly)
 - [good-winston](https://github.com/lancespeelmon/good-winston)
 - [hapi-good-logstash](https://github.com/atroo/hapi-good-logstash)
+
+## Reporter Interface
+
+When creating a custom good reporter, it needs to implement the following interface:
+- A constructor function (will always be invoked with `new`) with the following signature `function (events, config)` where:
+    - `events` - an object of key value pairs:
+        - `key` - one of the supported good events or any of the `extensions` events that this reporter should listen for
+        - `value` - a single string or an array of strings to filter incoming events. "\*" indicates no filtering. `null` and `undefined` are assumed to be "\*"
+    - `config` - an implementation specific configuration value used to instantiate the reporter
+- An `init` method with the following signature `function init (readstream, emitter, callback)` where:
+    - `readstream` - the incoming [readable stream](https://nodejs.org/api/stream.html#stream_class_stream_readable) from good. This stream is always in `objectMode`.
+    - `emitter` - the good event emitter. The `emitter` will emit the following events:
+        - `stop` - always emitted when the hapi server is shutting down. Perform any tear-down logic in this event handler
+
+```javascript
+var Squeeze = require('good-squeeze').Squeeze;
+var Fs = require('fs');
+
+var internals = {};
+internals.output = Fs.createWritableStream('somewhere/file.txt');
```

Comment:
Can't use internals here. If you do, ever instance of `GoodReporterExample` will point to 'somewhere/file.txt'. I would suggest just piping to standard out rather than introducing FS.